obspy/signal/tests/test_trigger.py

```diff
@@ -95,6 +95,17 @@ def test_pk_baer_cf(self):
         self.assertEqual(pfm, 'IPU0')
         self.assertEqual(len(cf), 119999)
 
+    def test_aic(self):
+        """
+        Test AIC function aganist implementation for UNESCO short course
+        """
+        filename = os.path.join(self.path, 'manz_waldk.a01.gz')
+        with gzip.open(filename) as f:
+            data = np.loadtxt(f, dtype=np.float32)
+        idx, aic_cf = aic(data)
+        self.assertEqual(idx, 60868)
+        self.assertEqual(len(aic_cf), 119999)
```

Comment:
maybe also test the values of the cf? compare other tests, e.g. with `np.testing.assert_array_almost_equal`

Reply:
test on cf added, see comment below